static/js/compose.js

```diff
@@ -1134,7 +1134,7 @@ $(function () {
             break;
         case 'FileTooLarge':
             // sanitization not needed as the file name is not potentially parsed as HTML, etc.
-            msg = "\"" + file.name + "\"" + i18n.t(" was too large; the maximum file size is 25MiB.");
+            msg = "\"" + file.name + "\" " + i18n.t("was too large; the maximum file size is 25MiB.");
```

Comment:
I'll weigh in here and agree with Umair, although I get Yago's point, too.  I think the variable gives the string more context for translators.  If there is risk of translators not respecting the double-underscore convention, then I think we need to address that separately.  Code readability is important, and the code that Umair proposes seems cleaner to me.